### PR TITLE
Feature rebased bipad

### PR DESCRIPTION
**Pull request by tnagorra**

### CODE REVIEW

File: components/Input/Checkbox/styles.scss [hunk 1]
```diff
@@ -3,6 +3,8 @@
     --width-checkbox-checkmark: var(--font-size-medium);
 }
 
+$width-checkmark: $font-size-medium;
```
Comment (tnagorra) [inline components/Input/Checkbox/styles.scss]: This looks broken

File: components/View/Taebul/Row/styles.scss [hunk 2]
```diff
@@ -5,6 +5,8 @@
 
     .row {
         display: flex;
+        height: 100%;
+        font-size: $font-size-medium;
```
Comment (tnagorra) [inline components/View/Taebul/Row/styles.scss]: This looks broken

File: components/Visualization/Legend/LegendItem/styles.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
+@import '~base-scss/base';
+
+.legend-item {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    .icon {
+        flex-shrink: 0;
+        margin-right: $spacing-small;
```
Comment (tnagorra) [inline components/Visualization/Legend/LegendItem/styles.scss]: This looks broken

File: components/Visualization/Map/MapDraw/index.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
+import React from 'react';
```
Comment (tnagorra) [inline components/Visualization/Map/MapDraw/index.js:1]: Is this useful?

File: components/Input/Checkbox/styles.scss (hunk 1 above)
Comment (frozenhelium) [reply components/Input/Checkbox/styles.scss]: Not anymore

File: components/Visualization/Legend/LegendItem/styles.scss (hunk 3 above)
Comment (frozenhelium) [reply components/Visualization/Legend/LegendItem/styles.scss]: Not anymore

File: components/Visualization/Map/MapDraw/index.js (hunk 4 above)
Comment (frozenhelium) [reply components/Visualization/Map/MapDraw/index.js:1]: Is it?

**Pull request merged**